### ISSUE TITLE
Support Nova 4[Feature Request]

### ISSUE THREAD

**Issue by newtec-foggia** [acvxqs/nova-registerpage#3]

**Comment by acvxqs** [comment]
I'm fairly new at anything Laravel + Laravel Nova. While I have a license for Nova 4, upgrading this package to work with it, is not currently on the list of priorities for me.

**Comment by junaryanto** [comment]
Nova 4 is so much better architecturally, it's big step with single page app.. People will migrate very soon

**Comment by denniselsinga** [comment]
News on this?

**Comment by acvxqs** [comment]
Nope, none. However there's a fork here which may work (It has a version bump in composer.json - I've not tried it): https://github.com/pjhile/nova-registerpage

**Comment by denniselsinga** [comment]
Thanks for heads up

**Comment by Mirmuxsin** [comment]
> composer require pjhile/nova-registerpage

it would work if you run `composer require acvxqs/nova-registerpage` after running `composer config repositories.nova-registerpage '{"type": "vcs", "url":"https://github.com/pjhile/nova-registerpage"}' --file composer.json`
